docs: add example to extract HEX encoded fingerprint from certificate (#33507)
elastic/beats: libbeat/docs/shared-ssl-config.asciidoc

`certificate_authorities` list and the handshake will continue
normaly.

One way to get the fingerprint from a CA certificate is using the

> following command where `ca.crt` is the certificate.

Reviewer: You may want to explicitly state that this applies to Unix like systems (Linux, Mac). It is obvious to us but users who only know Windows or are not very technical may read this as well.

Author: Done on [b63382e](https://github.com/elastic/beats/pull/33507/commits/b63382e966a7b8c11da018cad97e47ebde72d35c).

> One way to get the fingerprint from a CA certificate is, on a Unix like
> system, using the following command where `ca.crt` is the
> certificate.

Reviewer 2: ```suggestion
To get the fingerprint from a CA certificate on a Unix-like
system, you can use the following command, where `ca.crt` is the
certificate.
```
